http-filter-example/README.md

```diff
@@ -48,12 +48,40 @@ http_filters:
 - name: sample          # before envoy.router because order matters!
   typed_config:
     "@type": type.googleapis.com/sample.Decoder
-    key: via
+    key: Via
```

Comment:
nit: keep this lowercase

Reply:
This is just for testting this function
http_filter.cc
const LowerCaseString HttpSampleDecoderFilter::headerKey() const {
  return LowerCaseString(config_->key());
}